modules/select2_facets/src/Controller/FacetApiAutocompleteController.php

```diff
@@ -103,7 +76,7 @@ public function handleAutocomplete(Request $request, $facetsource_id, $facet_id,
 
       // Selection settings are passed in as a hashed key of a serialized array
       // stored in the key/value store.
-      $selection_settings = $this->keyValue->get($selection_settings_key, FALSE);
+      $selection_settings = $this->keyValue('entity_autocomplete')->get($selection_settings_key, FALSE);
```

Comment:
Wow the keyValue() implementation is funky - don't like the look of that - not your fault though.